src/server.c

```diff
@@ -5747,15 +3804,131 @@ void addReplyFlagsForKeyArgs(client *c, uint64_t flags) {
     setDeferredSetLen(c, flaglen, flagcount);
 }
 
-void addReplyCommandKeyArgs(client *c, struct redisCommand *cmd) {
+/* Must match redisCommandArgType */
```

Comment:
let's put a comment on the other end too.
same for the other enums below.

Reply:
ok

please note that currently both redisCommandRESP2Type and redisCommandRESP3Type are unused

should we keep them (it'll be helpful for https://github.com/redis/redis/issues/9845

Reply:
i guess we should trim them and add them in the later PR.
but i'm also ok with adding them now and use later.

Reply:
so i'll keep them